src/App.tsx

```diff
@@ -94,6 +104,7 @@ class App extends React.Component<Props, State> {
           <Route path="/tutorials" exact component={Tutorials} />
           <Route path="/scene/:sceneId" component={Root} />
           <Route path="/challenge/:challengeId" component={ChallengeRoot} />
+          <Route path="/curriculum" component={CurriculumPage} />
```

Comment:
A link to this needs to be added somewhere so that users have a way to get to it

Reply:
Happy to add it, but I was thinking we shouldn't add the button and make it visible until it is ready for launch. Do you have a preference for how and where to show this link?